keyboards/silverbullet44/keymaps/default/config.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright %YEAR% %YOUR_NAME%
```

Comment:
```suggestion
/* Copyright 2019 SwanMatch
```
You should update the copyright here with your name and year.